@xen-orchestra/web-core/lib/locales/en.json

```
@@ -53,6 +53,7 @@
   "no-data": "No data",
   "object-not-found": "Object {id} can't be found…",
   "other": "Other",
+  "page-not-found": "This page is not to be found…",
```

what your thought if we replace the sentence for This page cannot be found, it will be more idiomatic and revelant ?
```suggestion
  "page-not-found": "This page cannot be found…",
```

---

I agree with your comment. However, we will keep the current wording as it is used in the design system.
@clemencebx Any thoughts?
Since it is not a big change, I will merge this PR, and we could update the wording later.